drivers/platform/maxim/max32650/maxim_gpio_irq.c

```diff
@@ -106,6 +106,11 @@ void GPIO2_IRQHandler()
 }
 #endif
 
+void GPIO3_IRQHandler()
```

Comment:
needs `#ifdef MXC_GPIO3`

Reply:
not really, the max32650 has 4 GPIO ports. In fact, the other `ifdefs` should be removed.

Reply:
well, i picked a random one to comment on, but i think you got the point

Reply:
it seems to me easier to maintain if you copy paste code that covers the GPIO0 to GPIO3 range (or beyond) in all maxim_gpio_irq.c files without counting how many of them does the microcontroller really have